components/BottomSheet/src/private/MDCSheetContainerView.m

```diff
@@ -137,6 +145,25 @@ - (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
   }
 }
 
+- (void)safeAreaInsetsDidChange {
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    [super safeAreaInsetsDidChange];
+
+    _preferredSheetHeight = self.originalPreferredSheetHeight + self.safeAreaInsets.bottom;
+
+    UIEdgeInsets contentInset = self.sheet.scrollView.contentInset;
+    contentInset.bottom = MAX(contentInset.bottom, self.safeAreaInsets.bottom);
+    self.sheet.scrollView.contentInset = contentInset;
+
+    CGRect scrollViewFrame = self.sheet.scrollView.frame;
```

Comment:
Standardize before accessing the struct directly.
CGRect scrollViewFrame = CGRectStandardize(self.sheet.scrollView.frame);

Reply:
What's the reasoning behind that?

Reply:
We are directly accessing the struct directly below.
To ensure we are getting valid value out of a bounds / frame, we can either use the accessor functions like CGRectGetWidth() / Height() / etc. -OR- we can CGRectStandardize() the frame and access the standardized struct values directly.

Note that Swift eliminates all of this by only returning standardized values.  Our bright, safe Swifty future awaits! 😄